k8s_test.go

```diff
@@ -97,7 +97,7 @@ func (s *SuiteKubernetesCluster) Test_GetKubernetesNodepool() {
 	s.NotEmpty(np.Properties.StorageType)
 	s.NotEmpty(np.Properties.AutoScaling)
 	s.NotEmpty(np.Properties.MaintenanceWindow)
-	s.Len(np.Properties.PublicIPs, 3)
+	s.Len(*np.Properties.PublicIPs, 3)
```

Comment:
please wrap it by `if s.NotNil`

Reply:
@piepmatz  - was actually counting on the test data - I've made the change anyway. Thanks.